config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   root 'landing#index'
-  resources :photos, only: :index 
-end
+  resources :photos, only: :index
+
+  get '/search', to: 'photos#search'
```

Comment:
You can ignore this comment. I will update the config to not have this in future